package.json

```diff
@@ -138,6 +139,7 @@
     "prettier": "2.8.1",
     "puppeteer": "10.0.0",
     "quicktype-core": "6.1.0",
+    "react-textarea-autosize": "8.4.0",
```

Comment:
Adding these dependencies was to fix the storybook type errors right? Can you move these to a separate PR? The way our tsconfig/eslint is set up storybook type errors shouldn't effect builds or lintstaged. I'm pretty sure I asked Franco about adding these awhile ago so I want his eyes on the separate PR

Reply:
I don't know what this is. It was already a part of the code base.